doc/source/whatsnew/v1.2.1.rst

```diff
@@ -31,6 +31,7 @@ Fixed regressions
 - Fixed regression in :meth:`DataFrame.replace` raising ``ValueError`` when :class:`DataFrame` has dtype ``bytes`` (:issue:`38900`)
 - Fixed regression in :meth:`Series.fillna` that raised ``RecursionError`` with ``datetime64[ns, UTC]`` dtype (:issue:`38851`)
 - Fixed regression in comparisons between ``NaT`` and ``datetime.date`` objects incorrectly returning ``True`` (:issue:`39151`)
+- Fixed regression in calling NumPy ``accumulate`` ufuncs on DataFrames (:issue:`39259`)
```

Comment:
can you properly reference accumulate. i would be more clear about this note, and show and example `np.maximum.accumulate(df)` or similar (just the name no need to show computation)

Reply:
Do you mean adding a reference to the numpy docs? 
In case, I already added the example here.

Reply:
yes a refernce to the numpy docs

Reply:
OK, done